Makefile

```diff
@@ -240,6 +244,11 @@ docopen: out/doc/api/all.html
 docclean:
 	-rm -rf out/doc
 
+run-ci:
+	$(PYTHON) ./configure --without-snapshot $(CONFIG_FLAGS)
```

Comment:
Actually, I don't fully understand the implications on the download argument and whether it's a good idea to download the tarball in Jenkins. I will leave it to a follow-up PR as originally suggested.